test/controllers/sessions_controller_test.rb

```diff
@@ -44,6 +44,12 @@ class SessionsControllerTest < ActionDispatch::IntegrationTest
 
     assert_not_nil current_user
     assert_not_nil cookies[:remember_token]
+
+    remember_me_cookie = cookies.get_cookie("remember_token")
+
+    assert remember_me_cookie.http_only?
+    assert remember_me_cookie.secure?
+    assert_equal "Strict", remember_me_cookie.to_h["SameSite"]
```

Comment:
I'm surprised this passed, since it doesn't look like the implementation changed?

https://github.com/stevepolitodesign/rails-authentication-from-scratch/blob/b3e253fe2986d5672ba50f4ca23d4af038d5e8b1/app/controllers/concerns/authentication.rb#L37-L39

Reply:
Ah, it did not pass.

@mdchaney would you be able to make the implementation change too, as outlined in #53?

Reply:
Didn't pass.  I tested this in the wrong directory (long story).  I'm going to fix this.